lib/modules/manager/pip_requirements/extract.ts

```diff
@@ -111,6 +114,7 @@ export function extractPackageFile(
         depName,
         currentValue,
         datasource: PypiDatasource.id,
+        versioning: pep440Versioning.id,
```

Comment:
No need to set this explicitly as pypi datasource defaults to it 
```suggestion
```

Reply:
fixed